[IMP] core: new usage of Model.mapped() (#11295)
odoo/documentation: content/developer/reference/backend/orm/changelog.rst

- The `json` controllers have been renamed to `jsonrpc`. They are called the same, only the
  `type` in the python files changed. See `#183636 <https://github.com/odoo/odoo/pull/183636>`_.

> - Model's `mapped` now behaves more like the `map` function, it returns always a

Reviewer: If you referer to the class name:
```suggestion
- `Model`'s `mapped` now behaves more like the `map` function, it returns always a
```
If you meant "the method of models":
```suggestion
- Models' `mapped` now behaves more like the `map` function, it returns always a
```

> - Model's `mapped` now behaves more like the `map` function, it returns always a

Reviewer: ```suggestion
- Model's `mapped` now behaves more like the `map` function, it always returns a
```

> - Models' `mapped` now behaves more like the `map` function, it always returns a

Reviewer: ```suggestion
- Models' `mapped` function now behaves more like the `map` function, it always returns a
```

> - Models' `mapped` now behaves more like the `map` function, it always returns a

Reviewer: ```suggestion
- Models' `mapped` now behaves more like the `map` function: it always returns a
```

>   list of where each element of the recordset is mapped. See `#185630 <https://github.com/odoo/odoo/pull/185630>`_.

Reviewer: ```suggestion
  list where each element of the recordset is mapped. See `#185630 <https://github.com/odoo/odoo/pull/185630>`_.
```
